federated_query/plan/physical.py

```
@@ -838,15 +955,16 @@ def _scan_alias(self, scan: PhysicalScan) -> str:
-        alias = scan.alias
-        if scan.filters is None:
-            if alias:
-                return f"{table_ref} AS {alias}"
-            return table_ref
-        alias_name = alias if alias else scan.table_name
-        filter_sql = scan.filters.to_sql()
-        return f"(SELECT * FROM {table_ref} AS {alias_name} WHERE {filter_sql}) AS {alias_name}"
+        source_sql = table_ref
+        if scan.filters:
+            filter_sql = scan.filters.to_sql()
+            source_sql = f"(SELECT * FROM {table_ref} WHERE {filter_sql})"
```

Ruff (0.14.5)

Possible SQL injection vector through string-based query construction

(S608)